vendor/github.com/signalfx/signalfx-go/chart/model_options.go

```diff
@@ -31,6 +31,8 @@ type Options struct {
 	IncludeZero      bool              `json:"includeZero,omitempty"`
 	LegendOptions    *DataTableOptions `json:"legendOptions,omitempty"`
 	LineChartOptions *LineChartOptions `json:"lineChartOptions,omitempty"`
+	// Determines whether to hide missing data points in the chart. If `true`, missing data points in the chart would be hidden.
+	HideMissingValues bool `json:"hideMissingValues,omitempty"`
```

Comment:
This will be need to made upstream then the dependency reved: https://github.com/signalfx/signalfx-go

Reply:
Hi, can you please elaborate a bit on this? I'm not sure whether I got it right.

Reply:
This code is vendored from a go dependency. vendor code shouldn't be updated directly. You'll need to update the code in https://github.com/signalfx/signalfx-go, get it merged, update go.mod here, and run `go mod vendor` to update vendor.

Reply:
new version of vendor code (`v1.7.12`) has been updated in `go.mod` and ran `go mod vendor`